templates/header.war.html

```diff
@@ -11,12 +11,19 @@
 <p>
   You will need to pre-install a supported version of Java:
 <p>
-
-<ul>
-  <li>2.164 (2019-02) and newer: Java 8 or Java 11</li>
-  <li>2.54 (2017-04) and newer: Java 8</li>
-  <li>1.612 (2015-05) and newer: Java 7</li>
-</ul>
+      <b> LTS Version</b>
```

Comment:
If this is the first usage of the acronym "LTS", then the acronym should we expanded (I think):
```suggestion
          <b> <a href="https://www.jenkins.io/download/lts/">Long Term Support (LTS)</a> Version</b>
```

Reply:
makes sense and mirrors other change, will update